codewars9.py

```diff
@@ -0,0 +1,39 @@
+# https://www.codewars.com/kata/54e6533c92449cc251001667/train/python
+# I misunderstood the assignment and wrote code which only prints the first occurence of a character in a string.
+# I will repair it later.
+# def unique_in_order(iterable):
+
+iterable = 'AAAABBBCCDAABBB'
+iterable_new = []
+for index, x in enumerate(iterable):
+    if index in range (len(iterable)):
+        if iterable.count(x) > 1:
+            if iterable[index + 1] == '{}'.format(x):
```

Comment:
After `==` you can just use `x`, there's not much point in using .format notation if the only thing you want to compare it to is the value of `x`